srcpkgs/valadoc/template

```diff
@@ -0,0 +1,50 @@
+# Template file for 'valadoc'
```

Comment:
Maybe add a header here saying that this has to be kept in sync with vala (see chromium/firefox for now it's usually done)